pkg/providers/network.go

```diff
@@ -0,0 +1,13 @@
+package providers
+
+import (
+	"github.com/weaveworks/ignite/pkg/network/cni"
+)
+
+// NetworkPlugin provides the default network plugin implementation
+var NetworkPlugin cni.NetworkPlugin
+
+func SetCNINetworkPlugin() (err error) {
+	NetworkPlugin, err = cni.GetCNINetworkPlugin(Runtime)
```

Comment:
this is ok, but a bit misleading, as by default we don't use this provider in the codepath at all

Reply:
It's currently only used on one place when using CNI networking, but at least it has its own provider now. Might rename it later if it causes extra confusion.